tsdb/head.go

```diff
@@ -1451,17 +1456,17 @@ type memSeries struct {
 
 	ref           uint64
 	lset          labels.Labels
-	mmappedChunks []*mmappedChunk
-	mmMaxTime     int64 // Max time of any mmapped chunk, only used during WAL replay.
-	headChunk     *memChunk
+	mmappedChunks []*mmappedChunk // Immutable chunks on disk that have not yet gone into a block, in order of ascending time stamps.
+	mmMaxTime     int64           // Max time of any mmapped chunk, only used during WAL replay.
+	headChunk     *memChunk       // Most recent chunk in memory that's still being built.
 	chunkRange    int64
-	firstChunkID  int
+	firstChunkID  int // ChunkID corresponding to mmappedChunks[0].ref
```

Comment:
That's not true. It refers to the ChunkID for mmappedChunks[0], and mmappedChunks[1] being firstChunkID+1, and so on. With firstChunkID+len(mmappedChunks) being the head chunk.

Reply:
I think what you're saying is a superset of what i'm saying?
What's wrong with my comment? - can you make a suggestion?

Reply:
Not really. ChunkID referes to the id that corresponds to `mmappedChunks[0]`, and `firstChunkID+i` for `mmappedChunks[i]`, `mmappedChunks[0].ref` is the chunk reference used to access m-mapped chunk from the disk (the 4/4B packed ref).